presto-password-authenticators/src/test/java/io/prestosql/plugin/password/ldap/TestLdapEncoding.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.password.ldap;
+
+import org.testng.annotations.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestLdapEncoding
+{
+    @Test
+    void testDNEncoding()
+    {
+        // DNs which don't require encoding
+        assertThat(LdapAuthenticator.encodeForDN("I can eat glass and it doesn't hurt me"))
```

Comment:
This text is a bit strange (some may find it offensive). Let’s use something more generic.